stories/list-grid/list-grid.stories.js

```diff
@@ -0,0 +1,1880 @@
+                    </div>
+                </div>
+            </div>
+
+            <div  class="fd-col fd-col--12 fd-col-md--6 fd-col-lg--4 fd-col-xl--3">
+                <div tabindex="0" class="fd-grid-list__item">
+                    <div class="fd-grid-list__item--body fd-grid-list__item--body--padding">
+                        <div class="grid-list-item-body-example">
+                            <span class="fd-avatar fd-avatar--s grid-list-item-body-example-avatar" role="presentation" aria-label="John Doe"></span>
+                            <div>
+                                <h4 class="fd-title fd-title--h4">John Li</h4>
+                                <p>Product Owner, Company B</p>
+
+                                <div>
+                                    <p>781 Main Street</p>
+                                    <p>Anytown, SD 57401</p>
+                                    <p>USA</p>
+                                </div>
+
+                                <a href="#" class="fd-link" tabindex="0">john_li@example.com</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+noneMode.storyName = 'None mode';
+
+noneMode.parameters = {
+    docs: {
+        storyDescription: 'Items cannot be selected but can still use "navigation", which allows click handling on specific items.'
+    }
+};
+
+export const singleSelectMasterMode = () => `<div class="fd-grid-list fd-grid-list__mode--single-select-master">
+    <div class="fd-container">
+        <div class="fd-row">
+            <div class="fd-col fd-col--12">
+                <div class="fd-toolbar fd-toolbar--solid">
+                    <span>Products (420)</span>
+
+                    <span class="fd-toolbar__spacer"></span>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Pool">
+                        <i class="sap-icon--sort"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Chart">
+                        <i class="sap-icon--filter"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Chart">
+                        <i class="sap-icon--group-2"></i>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="fd-row">
+            <div class="fd-col fd-col--12 fd-col-md--6 fd-col-lg--4 fd-col-xl--3">
+                <div tabindex="0" class="fd-grid-list__item fd-grid-list__item--link">
+                    <input type="radio" class="fd-radio fd-radio--compact fd-grid-list__radio--input" id="pDidh761" name="singleSelectMasterRadio">
+                    <label class="fd-radio__label fd-grid-list__radio--label" for="pDidh761"></label>
```

Comment:
This should not be an empty label. At the moment when you focus the radio button, the entire card reads out. There should be some sort of label to summarize what the item is (title, etc)

Reply:
fixed